CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.2.0
+
+## Features
+
+- Auto preview on fail (TODO: Polish this changelog)
```

Comment:
@nvh95 are we polishing this change log in this PR, or leave it for a future PR?

Reply:
Just a note to polish the changelog since it's a bit late when I was committing it :P